arch/arm64/start.S

```diff
@@ -94,6 +100,11 @@ FUNCTION(_start)
 .Lnot_device:
 
 .Lmapping_size_loop:
+    ldr     attr, =MMU_PTE_KERNEL_DATA_FLAGS
+    ldr     tmp, =arm_reset
```

Comment:
might be nitpicky, but we might want to use __code_start here instead of arm_reset.